packages/components/svgoUtils.js

```diff
@@ -44,13 +44,17 @@ const replaceAttrKeys = child => {
 // Figma/ Sketch don't allow us to set 'currentColor' when exporting SVGs.
 // We have a convention to export the color as #000 or #0000000, then change it here.
 const replaceColor = child => {
-  if (child.attributes.fill === "#000" || child.attributes.fill === "#000000") {
+  const hexCodesToReplace = [
+    "#000",
+    "#000000",
+    "#2F2438", // Purple-800 from Figma export
+  ]
+
+  if (hexCodesToReplace.includes(child.attributes.fill)) {
```

Comment:
Good stuff, just wish JS read better in an english sense lol ie. "fill includes a hex code to replace"

Reply:
Yeah I actually had to double take to ensure this wasn't changing the condition 😅